src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -175,7 +175,7 @@ public static void TryCompileScript(string script)
             {
                 var engine = new Engine(options =>
                 {
-                    options.MaxStatements(1).LimitRecursion(1);
+                    options.MaxStatements(128).LimitRecursion(1);
```

Comment:
https://issues.hibernatingrhinos.com/issue/RavenDB-24767/Failed-to-export-documents-with-transform-script#focus=Comments-67-1066881.0-0